plugins/amazonq/codewhisperer/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codewhisperer/language/CodeWhispererProgrammingLanguage.kt

```diff
@@ -18,7 +16,6 @@ import software.aws.toolkits.telemetry.CodewhispererLanguage
  */
 abstract class CodeWhispererProgrammingLanguage {
```

Comment:
do we still need this concept? or should we let the language server decide itself if the language is supported

Reply:
no but this will unveil a discrepancy between IDE and flare implementation on language support but we are tracking it anyway, will remove this.

Reply:
also language server only returns EMPTY_RESULTS on unsupported messages while IDE display some error hints after manual trigger on an unsupported file

Reply:
also codesacan might still need it, I will remove just the inline aspect of it.